src/hotspot/share/cds/filemap.cpp

```diff
@@ -289,7 +289,11 @@ void FileMapHeader::print(outputStream* st) {
   st->print_cr("- has_non_jar_in_classpath:       %d", _has_non_jar_in_classpath);
   st->print_cr("- requested_base_address:         " INTPTR_FORMAT, p2i(_requested_base_address));
   st->print_cr("- mapped_base_address:            " INTPTR_FORMAT, p2i(_mapped_base_address));
-  st->print_cr("- heap_roots_offset:              " SIZE_FORMAT, _heap_roots_offset);
+  st->print_cr("- heap_root_segments.roots_count: %d" , _heap_root_segments.roots_count());
+  st->print_cr("- heap_root_segments.base_offset: " SIZE_FORMAT, _heap_root_segments.base_offset());
```

Comment:
Existing offset value are being printed with the format SIZE_FORMAT_X. Maybe use the same format here.

Reply:
Right, this should remain in hex. Fixed.